examples/ch32v003/README.md

```diff
@@ -22,3 +22,21 @@ Link: [OSHWHUB](https://oshwhub.com/andelf/linedog)
 
 - BUZZ: PC4 (TIM1CH4)
 - WS2812: PA2
+
+## Running Examples
+
+### Using [wlink](https://github.com/ch32-rs/wlink)
+- Install wlink using the installation instructions: https://github.com/ch32-rs/wlink?tab=readme-ov-file#install
+
+- Edit the [`.cargo/config.toml`](.cargo/config.toml) file so the runner is `wlink`. This may already be the default runner.
+
+- Build and run the [blinky](src/bin/blinky.rs) example with `cargo run --release --bin blinky`.
+
+### Using [probe-rs](https://probe.rs/)
+
+- Install probe-rs using the installation instructions: https://probe.rs/docs/getting-started/installation/
+    - If you are on a Linux based system, you may have to add udev rules to allow probe-rs access to the WCH-Link debugger. https://probe.rs/docs/getting-started/probe-setup/#linux%3A-udev-rules
```

Comment:
I actually had to modify the udev rules. My WCH-Link shows up with lsusb as:
`Bus 003 Device 033: ID 1a86:8010 QinHeng Electronics WCH-Link`
which doesn't seem to match the usual rule with an id of 1a86:8011 from the rules on the probe-rs docs:
https://probe.rs/docs/getting-started/probe-setup/#linux%3A-udev-rules

Should I open a pull request with probe-rs to add that?